src/script/conversation/ConversationFilter.ts

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
WDYT about putting this into the `UserState` or `ConversationState`?

Reply:
The `ConversationFilter` has only stateless functions so its functions are not a good fit for the `ConversationState` which is stateful.